govdelivery-subscribe/error/index.html

```diff
@@ -11,13 +11,13 @@
 {% block content_main %}
     <h1><span class="cf-icon cf-icon-error-round"></span> Not just yet!</h1>
     <p class="short-desc">
-        We’re still building the email signup function for <br>
-        this beta release. It will be available soon. For now:
+        We couldn’t complete your signup request. Try<br>
```

Comment:
Is disabling the Sign up button till a user has entered an email address an option? Is there any reason to allow a user to click that button and go to an error page?